extensions/smallrye-context-propagation/spi/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-smallrye-context-propagation-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-smallrye-context-propagation-deployment-spi</artifactId>
```

Comment:
This should be called just `quarkus-smallrye-context-propagation-spi`, to follow the naming convention of all other `spi` artifacts.

I used this command to see the existing convention:

```bash
$ for D in $(find . -type d -name spi | grep -v "src/main" | grep -v "src/test") ; do grep "<artifactId>" $D/pom.xml | grep spi ; done
```

Outcome:

```
    <artifactId>quarkus-smallrye-metrics-spi</artifactId>
    <artifactId>quarkus-kubernetes-service-binding-spi</artifactId>
    <artifactId>quarkus-container-image-spi</artifactId>
    <artifactId>quarkus-smallrye-health-spi</artifactId>
    <artifactId>quarkus-smallrye-openapi-spi</artifactId>
    <artifactId>quarkus-jsonb-spi</artifactId>
    <artifactId>quarkus-kubernetes-client-spi</artifactId>
    <artifactId>quarkus-agroal-spi</artifactId>
    <artifactId>quarkus-hibernate-validator-spi</artifactId>
    <artifactId>quarkus-security-spi</artifactId>
    <artifactId>quarkus-undertow-spi</artifactId>
    <artifactId>quarkus-kubernetes-spi</artifactId>
    <artifactId>quarkus-resteasy-server-common-spi</artifactId>
    <artifactId>quarkus-resteasy-common-spi</artifactId>
    <artifactId>quarkus-jackson-spi</artifactId>
```

Reply:
Ok, let's rename it. I was looking at [quarkus-vertx-http-deployment-spi](https://github.com/quarkusio/quarkus/blob/main/extensions/vertx-http/deployment-spi/pom.xml#L13) which has the `-deployment` part because thre's also a `dev-console-spi`...

Reply:
Ah, didn't know about that. Thanks for info!